.editorconfig

```diff
@@ -0,0 +1,7 @@
+[*.swift]
+indent_style = space
+indent_size = 4
+tab_width = 4
```

Comment:
Nitpicking here, so ignore if you think otherwise. This seems to default to `indent_size`, so the line is not needed. But maybe you have your reasons. I am always curious.

Reply:
Yes, can be removed. I just like to be explicit usually, instead of implicit.

Reply:
No, keep it than :-)